tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -140,24 +140,26 @@ def run_cmd(self, callback, *args, **kwargs):
 
         return call_result
 
-    def init(self):
+    def init(self, log_files=None):
```

Comment:
Do we really need to define and use a new argument `log_files` here? Can we not somehow work with existing `self.additional_files`?

Reply:
Using `self.additional_files` for log files to initialize.